src/subcommand/server.rs

```diff
@@ -832,6 +837,75 @@ impl Server {
     })
   }
 
+  async fn inscription_details(
```

Comment:
Maybe call this `inscription_recursive` so it's clear that this function has stricter guarantees than the normal server functions.

Reply:
Fixed in debb433585e50e3e3f2615204c6984d5e702db9c